tools/Utilities/src/Views/UtilityView.xaml

```diff
@@ -57,29 +59,24 @@
             </StackPanel>
         </ScrollViewer>
 
-        <SplitButton 
+        <ToggleSwitch
             Grid.Row="2"
-            HorizontalAlignment="Center"
+            IsOn="{x:Bind ViewModel.LaunchAsAdmin, Mode=TwoWay}"
+            Visibility="{x:Bind ViewModel.LaunchAsAdminVisibility, Mode=OneWay}"
+            OnContent="Launch as administrator"

```

Comment:
Both On and Off use the same content?  This tells me that the program will still launch as admin even when toggled off.

Reply:
To me, it is like - turn it on to launch as admin. i.e. to me, the text doesn't represent the current state of the switch but what would happen if turn it on.